lib/url.c

```diff
@@ -3598,14 +3611,14 @@ static CURLcode create_conn(struct Curl_easy *data,
     *in_connect = conn;
 
 #ifndef CURL_DISABLE_PROXY
-    infof(data, "Re-using existing connection with %s %s",
-          conn->bits.proxy ? "proxy" : "host",
+    infof(data, "Re-using existing %s: connection with %s %s",
+          conn->handler->scheme, conn->bits.proxy ? "proxy" : "host",
```

Comment:
minor nit I suggest get rid of the scheme colon or revert to the original text for this and the next one. It may be confusing to say IMAP connection reused when the user may be expecting IMAPS?

Reply:
But it would reflect what is happening. The only information missing is that the imap: connected has TLS. Mabye we should add that?

Reply:
I think it would be helpful to add that, yes

Reply:
Added information about SSL upgrade status of the reused connection.